test/mock/service.go

```diff
@@ -79,6 +115,9 @@ func MakeInstance(service *model.Service, port *model.Port, version int) *model.
 // MakeIP creates a fake IP address for a service and instance version
 func MakeIP(service *model.Service, version int) string {
 	ip := net.ParseIP(service.Address).To4()
+	if ip == nil {
```

Comment:
I think this will be the case until we start using Endpoints for external services